<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bayesian working platform</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background:rgb(255, 255, 255);
	font-size:15px;
	font-family:Century Gothic,sans-serif;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background:rgb(255, 255, 255);
font-size:15px;
font-family:Century Gothic,sans-serif;
}
QFrame{
border:sold 10px rgba(0,0,0);
background:darkgray;
}
QLineEdit{
color:#8d98a1;
background-color:#405361;
font-size:16px;
border-style:outset;
border-radius:10px;
}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
}
QPushButton:pressed{
background-color:rgb(224,0,0);
border-style:inset;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:italic bold;
font-size:14px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>119</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remember me</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Forget password?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Academy Engraved LET&quot;
</string>
    </property>
    <property name="text">
     <string>Bayesian Network analysis</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
